### ISSUE TITLE
error when reverting a single changeset

### ISSUE THREAD

**Issue by maning** [Zverik/simple-revert#9]
I get this:

```
./simple-revert.py 37139422
Downloading changeset 37139422Traceback (most recent call last):
  File "./simple-revert.py", line 189, in <module>
    obj = obj_to_dict(obj_xml)
  File "/Users/maning/projects/git/simple-revert/common.py", line 68, in obj_to_dict
    res['tags'] = { tag.get('k') : tag.get('v') for tag in obj.iterchildren('tag')}
AttributeError: iterchildren

```

**Comment by Zverik** [comment]
Right, I didn't check how a fall-back import of ElementTree module works. Fixed in 82c02e870227535968a9f8afe3026d5d258fe62d, thanks for testing.

**Comment by maning** [comment]
tnx! works now.
